### PR TITLE
Added a project and documentation folders.

### PR DESCRIPTION
**Pull request by KovalenkoAndrii**
Create a repo for application on GitHub

### CODE REVIEW

**Comment by TanyaKalistruk** [comment]
How about adding .gitignore for filtering files and not upload files with cashe or something like that?

File: MyBooking.Domain/Entities/Advert.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace MyBooking.Domain.Entities
+{
+    public class Advert
+    {
+        public int Id { get; set; }
+        public DateTime DateCreated { get; set; } = DateTime.Now;
+        public DateTime? LastModifyDate { get; set; } = null;
```
Comment (TanyaKalistruk) [inline MyBooking.Domain/Entities/Advert.cs]: What for there is '?' at the end of DateTime?

File: MyBooking.Domain/Interfaces/IAdvertRepository.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+
+namespace MyBooking.Domain.Interfaces
+{
+    public interface IAdvertRepository
+    {
+        List<Advert> GetAll();
+        Advert GetById(int id);
+        Advert GetByAddress(string address);
+        List<Advert> GetWithFilters(bool animals = false, int countPeople = 0, int minCost = 0, int maxCost = 0);
```
Comment (TanyaKalistruk) [inline MyBooking.Domain/Interfaces/IAdvertRepository.cs:11]: How about making default value for count of people 1? 0 is strange...

File: MyBooking.Repositories/MyBooking.Repositories.csproj [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (TanyaKalistruk) [inline MyBooking.Repositories/MyBooking.Repositories.csproj:1]: What does this file do?

File: MyBooking.Repositories/RoleRepository.cs [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                .ToList();
+        }
+
+        public Role GetById(int roleId)
+        {
+            return Entities
+                .FirstOrDefault(f => f.Id == roleId);
+        }
+
+        public Role Insert(Role newRole)
```
Comment (TanyaKalistruk) [inline MyBooking.Repositories/RoleRepository.cs:34]: Do we need the option of adding new roles?

File: MyBooking.Repositories/RoleRepository.cs [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            }
+
+            Entities.Add(newRole);
+
+            SaveChanges();
+
+            return newRole;
+        }
+
+        public Role Update(int roleId, Role role)
```
Comment (TanyaKalistruk) [inline MyBooking.Repositories/RoleRepository.cs:48]: The same with update

File: MyBooking.sln [hunk 6]
```diff
@@ -0,0 +1,37 @@
+﻿
```
Comment (TanyaKalistruk) [inline MyBooking.sln:1]: Do we actually need this file?

File: MyBooking/Controllers/AdvertController.cs [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+    public class AdvertController : ControllerBase
+    {
+        private readonly IAdvertRepository advertRepository;
+
+        public AdvertController(IAdvertRepository advertRepository)
+        {
+            this.advertRepository = advertRepository;
+        }
+
+        /// GET Api/Advert
```
Comment (TanyaKalistruk) [inline MyBooking/Controllers/AdvertController.cs:20]: How about lowercase in URLs?

File: MyBooking.Repositories/MyBooking.Repositories.csproj (hunk 3 above)
Comment (KovalenkoAndrii) [reply MyBooking.Repositories/MyBooking.Repositories.csproj:1]: A software development kit (SDK) is a collection of software development tools in one installable package.

File: MyBooking.Repositories/RoleRepository.cs (hunk 4 above)
Comment (KovalenkoAndrii) [reply MyBooking.Repositories/RoleRepository.cs:34]: Yes it is necessary at this level

File: MyBooking.Repositories/RoleRepository.cs (hunk 5 above)
Comment (KovalenkoAndrii) [reply MyBooking.Repositories/RoleRepository.cs:48]: ^^

Comment (KovalenkoAndrii) [reply MyBooking.Repositories/RoleRepository.cs:48]: ^^

File: MyBooking.sln (hunk 6 above)
Comment (KovalenkoAndrii) [reply MyBooking.sln:1]: This is our solution file.

Comment (KovalenkoAndrii) [reply MyBooking.sln:1]: Yes we need this file.

File: MyBooking.Domain/Interfaces/IAdvertRepository.cs (hunk 2 above)
Comment (KovalenkoAndrii) [reply MyBooking.Domain/Interfaces/IAdvertRepository.cs:11]: This type of value means that this argument can be skipped, if 0, then this filter is not used

File: MyBooking/Controllers/AdvertController.cs (hunk 7 above)
Comment (KovalenkoAndrii) [reply MyBooking/Controllers/AdvertController.cs:20]: This is default value.

File: MyBooking.Domain/Interfaces/IAdvertRepository.cs (hunk 2 above)
Comment (TanyaKalistruk) [reply MyBooking.Domain/Interfaces/IAdvertRepository.cs:11]: Okay

File: MyBooking.Repositories/MyBooking.Repositories.csproj (hunk 3 above)
Comment (TanyaKalistruk) [reply MyBooking.Repositories/MyBooking.Repositories.csproj:1]: Do we actually need it in the repo?

File: MyBooking.Repositories/RoleRepository.cs (hunk 4 above)
Comment (TanyaKalistruk) [reply MyBooking.Repositories/RoleRepository.cs:34]: Okay, let it be

File: MyBooking.Repositories/MyBooking.Repositories.csproj (hunk 3 above)
Comment (KovalenkoAndrii) [reply MyBooking.Repositories/MyBooking.Repositories.csproj:1]: I think it’s possible to simplify the launch of the project.

Comment (TanyaKalistruk) [reply MyBooking.Repositories/MyBooking.Repositories.csproj:1]: Discussed

**Review by TanyaKalistruk — APPROVED** [review]

**Pull request merged**